community/ctags/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+./configure --disable-etags --prefix=$1/usr
```

Comment:
The format is:

```
./configure \
    --prefix=/usr \
    --disable-etags

make DESTDIR="$1" install
```

Reply:
I put each flag on its own line but DESTDIR isn't an option in the makefile which is why "$1" is part of the prefix